fk7263/src/main/java/se/inera/intyg/common/fk7263/model/converter/RespConstants.java

```diff
@@ -0,0 +1,124 @@
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_CATEGORY_ID = "arbetslivsinriktadRehabilitering";
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_CATEGORY_TEXT_ID = "fk7263.label.workrehab";
+    public static final String ARBETSLIVSINRIKTAD_REHABILITERING_SVAR_ID = "10";
+    public static final String PATIENTENS_ARBETFORMAGA_CATEGORY_ID = "patientensArbetsformoga";
+    public static final String PATIENTENS_ARBETFORMAGA_CATEGORY_TEXT_ID = "fk7263.label.patientworkcapacity";
+    public static final String PATIENTENS_ARBETFORMAGA_SVAR_ID = "11";
+    public static final String PATIENTENS_ARBETFORMAGA_NUVARANDE_ARBETE_TEXT_ID = "fk7263.label.patientworkcapacity.currentwork";
+    public static final String PATIENTENS_ARBETFORMAGA_FORALDRALEDIGHET_TEXT_ID = "fk7263.label.patientworkcapacity.parentalleave";
+    public static final String PATIENTENS_ARBETFORMAGA_ARBETSLOSHET_TEXT_ID = "fk7263.label.patientworkcapacity.unemployed";
+    public static final String ARBETFORMAGA_BEDOMNING_CATEGORY_ID = "arbetsformogaBedomning";
+    public static final String ARBETFORMAGA_BEDOMNING_CATEGORY_TEXT_ID = "fk7263.label.nedsattning.falt8b";
+
+    public static final String ARBETFORMAGA_BEDOMNING_SVAR_ID = "12";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_ID = "nedsatt";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_TEXT = "Nedsatt med";
+
+    public static final String ARBETFORMAGA_BEDOMNING_FOM_ID = "franOchMed";
+    public static final String ARBETFORMAGA_BEDOMNING_FOM_TEXT = "Från och med";
+
+    public static final String ARBETFORMAGA_BEDOMNING_TOM_ID = "tillOchMed";
+    public static final String ARBETFORMAGA_BEDOMNING_TOM_TEXT = "Längst till och med";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_25_TEXT_ID = "fk7263.nedsattningsgrad.nedsatt_med_1_4";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_50_TEXT_ID = "fk7263.nedsattningsgrad.nedsatt_med_1_2";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_75_TEXT_ID = "fk7263.nedsattningsgrad.nedsatt_med_3_4";
+    public static final String ARBETFORMAGA_BEDOMNING_NEDSATT_100_TEXT_ID = "fk7263.nedsattningsgrad.helt_nedsatt";
+
+    public static final String ARBETFORMAGA_FORSAKRINGSMEDICINSKA_BESLUTSTODET_CATEGORY_ID = "arbetsformogaForsakringsmedicinska";
+    public static final String ARBETFORMAGA_FORSAKRINGSMEDICINSKA_BESLUTSTODET_CATEGORY_TEXT_ID =
+        "fk7263.label.patientworkcapacityjudgement";
+    public static final String ARBETFORMAGA_FORSAKRINGSMEDICINSKA_BESLUTSTODET_SVAR_ID = "13";
+    public static final String ARBETFORMAGA_PROGRNOS_CATEGORY_ID = "arbetsformogaPrognos";
+    public static final String ARBETFORMAGA_PROGRNOS_CATEGORY_TEXT_ID = "fk7263.label.prognosis";
+
+    public static final String ARBETFORMAGA_PROGRNOS_SVAR_ID = "14";
+    public static final String RESOR_TILL_FRAN_ARBETE_CATEGORY_ID = "resorTillFranArbete";
```

Comment:
In all of the above, refactor ARBETFORMAGA->ARBETSFORMAGA and PROGRNOS -> PROGNOS

Reply:
Nice catch, fixed!